### PR TITLE
make tag names consistent with datadog

### PR DESCRIPTION
**Pull request by georgicodes**
We identified the need to have consistent tag names and hierarchy. And to have the tags be as close to the DD ones as possible.

### CODE REVIEW

**Comment by georgicodes** [comment]
@janester and @jnunemaker. This is my first foray into Lightstep! Is this what you had in mind in terms of making the tags be consistent with dd?

File: lib/hubstep/faraday/middleware.rb [hunk 1]
```diff
@@ -43,12 +43,12 @@ def record_request(span, request_env)
         method = request_env[:method].to_s.upcase
         span.operation_name = "Faraday #{method}"
         span.set_tag("component", "faraday")
-        span.set_tag("http.url", request_env[:url])
-        span.set_tag("http.method", method)
+        span.set_tag("url", request_env[:url])
+        span.set_tag("method", method)
```
Comment (aroben) [inline lib/hubstep/faraday/middleware.rb]: FYI, these tag names are [specified by OpenTracing](https://github.com/opentracing/specification/blob/22abcdb385a8282c4e9ce760beacdc9e8f13fe00/semantic_conventions.md). You can of course change them, I just thought you should be aware.

**Comment by jnunemaker** [comment]
I wasn't clear in the task I had on our issue. Sorry about that! These are actually all fine as they are and I agree with @aroben that we should stick with open tracing specifications. I'll hit you up in chat or our issue with more specifics as to what I was thinking.

**Comment by georgicodes** [comment]
Closing as this is not the right move here 🤘

**Comment by georgicodes** [comment]
@jnunemaker all good! i thought it was easier to push up something and get feedback even if it wasn't right.

**Pull request closed without merging**